mozilla/fx-private-relay
# Add announcement area for new features (React)

# New feature description

This adds an announcement area for new features, the ability to mark them as read, as well as getting back to previously-read announcements. It is the React implementation of https://github.com/mozilla/fx-private-relay/pull/1561.

(This also happens to fix #1700.)

# Screenshot (if applicable)

https://user-images.githubusercontent.com/4251/154990215-2921f62f-3f4f-464f-9f92-81da7915fd0b.mp4

# How to test

Same steps as in #1561. Additionally, verify that you can see previously-read entries in the history, and that you can move all entries to the history by selecting "Clear all". And ideally, also that everything is keyboard-accessible and properly announced by a screen reader.

# Checklist

- [ ] l10n dependencies have been merged, if any. - No, content is not final yet.
